src/Microsoft.Azure.WebJobs.Extensions.EventHubs/EventHubWebJobsBuilderExtensions.cs

```diff
@@ -12,7 +13,9 @@ public static class EventHubWebJobsBuilderExtensions
     {
         public static IWebJobsBuilder AddEventHubs(this IWebJobsBuilder builder)
         {
-            builder.AddExtension<EventHubExtensionConfigProvider>();
+            builder.AddExtension<EventHubExtensionConfigProvider>()
+                .BindOptions<EventProcessorOptions>();
```

Comment:
This is an example of using `BindOptions` to automatically bind to the extension section.